src/siema.js

```diff
@@ -56,8 +56,11 @@ export default class Siema {
       draggable: true,
       threshold: 20,
       loop: false,
-      onInit: function() {},
-      onChange: function() {},
+      onInit: () => {},
+      onChange: () => {},
+      afterPrev: () => {},
+      afterNext: () => {},
+      onDestroy: () => {}
```

Comment:
We have to ditch afterPrev() and afterNext() from this PR. In current implementation it invokes actually straight before next() and prev(), not after it. Doing before and after is a bit more complicated as it needs to listen for transitionstart and transitionend events on instance's sliderFrame. Let's keep it for future versions of Siema.

The onDestory() callback — perfecto :)